crest/Assets/Crest/Crest/Scripts/LodData/OceanDepthCache.cs

```diff
@@ -77,11 +78,26 @@ public enum OceanDepthCacheRefreshMode
         bool _runValidationOnStart = true;
 #pragma warning restore 414
 
-        RenderTexture _cacheTexture;
-        public RenderTexture CacheTexture => _cacheTexture;
+        [Header("Signed Distance Field (Experimental)")]
+        [Tooltip("Generate a signed distance field (experimental)"), SerializeField]
+        internal bool _generateSDF = false;
 
-        GameObject _drawCacheQuad;
-        Camera _camDepthCache;
+        [PredicatedField("_generateSDF")]
+        [Tooltip("How many additional Jump Flood Algorithm rounds to use - (over the standard log2(Resolution)"), SerializeField]
```

Comment:
Yes it does - but jumpflood isn't technically an accurate algorithm - it's result is an approximiation that trends towards the actual SDF. Additional rounds of JFA can help correct for some of these innacuracies. It's described in the paper. One sec...

Reply:
> We  observe  in  our  experiments  that  most  grid  points  with  errors are grid points clustered around Voronoi vertices, or around Voronoi  edges  intersecting  the  boundary  of  the  grid.  Moreover,  many  grid  points  with  errors  are  each  a  single  error  and  itself  a  Voronoi  vertex.  As  such,  a  simple  yet  effective  way  to  remove  many  errors  is  to  execute  JFA  for  one  additional  round  with  step  length  of  1,  i.e.  a  total  of  (logn)+1  rounds  where  both  of  the  last  two rounds are with step length of 1. We call this variant JFA+1.   It is easy to realize that one can then use JFA+2 which is the usual JFA plus two additional rounds, with step length of 2 and 1 respectively.  Likewise,  one  can  do  as  many  additional  rounds  as  needed  to  achieve  good  accuracy  (while  converging  to  no  better  than  the  standard  flood).  We  have  also  used  JFA  plus  lognadditional rounds, i.e. JFA+logn or simply JFA2. Our experiments as reported in Section 6 confirm the effectiveness of these variants. In  particular,  JFA2  produces  correct  results  in  most  cases,  though  it  fails  for  the  difficult  case  as  shown  in  Figure  5.  (This  is  also  a  case  where  [Borgefors1984]  cannot  correctly  handle  with  a  3×3  mask.) In this case, if the center grid point p is incorrect at the end of JFA, it will still remain incorrect at the end of JFA2. This is so as the Voronoi cell (shown as a wedge) does not “cover” any grid points along the dotted lines that p refers to during the additional logn  rounds,  i.e.  p  thus  never  receives  the  correct  seed  from  any  grid points of its Voronoi cell. 

https://www.comp.nus.edu.sg/~tants/jfa/i3d06.pdf

Reply:
That's really interesting thanks